.gitignore

```diff
@@ -35,3 +35,5 @@ yarn-error.log*
 
 # vscode
 .vscode
+
+apps/monk-test-app/build
```

Comment:
instead of having this live in the gitignore at the root of the repo, add it to the .gitignore file that is located in `apps/monk-test-app` (if there is no gitignore, you can create one)

Reply:
Done